### ISSUE TITLE
BUG: Linux tail -r does not work

### ISSUE THREAD

**Issue by SystemFiles** [SystemFiles/GOG#15]
As stated in https://github.com/mapmeld/gitjk/issues/3 , it would appear that while `tail -r` works on Mac OSX, it does not work on Linux distributions.

Resulting in the following error when trying to perform `gog finish -major`:

```
tail: invalid option -- 'r'
```

**Comment by SystemFiles** [comment]
In the article it was suggested that you can use `tac` and this does indeed work on Linux, but then breaks compatibility with Mac OSX unless explicitly installed. Instead, I've opted for the following solution:

```go
latestTag := [3]int{0,0,0}
	tagScanner := bufio.NewScanner(strings.NewReader(string(tagOut)))
	for tagScanner.Scan() {
		tag := tagScanner.Text()
		if matched := semverRegex.MatchString(tag); matched {
			semverTag, err := semver.Parse(tag)
			if err != nil {
				return version, err
			}

			if semverTag.GreaterThan(latestTag) {
				latestTag = semverTag
			}
		}
	}
```

**Comment by SystemFiles** [comment]
Fixed in v0.3.5
